src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/X86/AddCarry.PlatformNotSupported.cs

```diff
@@ -0,0 +1,41 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+using System;
+using System.Runtime.CompilerServices;
+
+namespace System.Runtime.Intrinsics.X86
+{
+    /// <summary>
+    /// This class provides access to Intel ADDCARRY hardware instructions via intrinsics
```

Comment:
nit: We should use the agnostic `x86` rather than `Intel` or `AMD`; as using a specific manufacturer leads to confusion about where this is supported.

Reply:
(I realize this is something that needs to be fixed on the other ISAs as well)